lib/bundler/rubygems_integration.rb

```diff
@@ -795,6 +802,12 @@ def use_gemdeps(gemfile)
         activated_spec_names = runtime.requested_specs.map(&:to_spec).sort_by(&:name)
         [Gemdeps.new(runtime), activated_spec_names]
       end
+
+      if provides?(">= 2.5.2")
```

Comment:
I thought it was rather self explanitory -- on 2.5.2+, bindstubs don't call `gem`. What sort of comment were you looking for?